### PR TITLE
feat(protocol-designer): Remove custom tiprack feature flag

### LINKED ISSUE #5441: PD: Remove Custom Tiprack FF and Update Tests
This ticket is a placeholder to remind us to remove the FF once feature work is complete and tested, and update tests to reflect the new UI changes by default.

### PR DESCRIPTION
**Pull request by shlokamin**
## overview
This PR closes #5441 by removing the custom tiprack upload FF. Custom tiprack upload is now a full feature, woo! 

## changelog
  - Remove custom tiprack upload FF
  - Update unit tests
  - Deleted unused `Tiprack Diagram` component

## review requests
I've never deleted a FF in PD before so please sanity check that I did everything I was supposed to...

 - Custom tiprack FF should no longer appear in settings
 - Should be able to upload custom tipracks in file pipette modal

## risk assessment
Low, just removing a FF for a tested feature

### CODE REVIEW

File: protocol-designer/src/components/modals/FilePipettesModal/__tests__/PipetteFields.test.js [hunk 1]
```diff
@@ -157,22 +154,6 @@ describe('PipetteFields', () => {
 
     expect(props.onFieldChange).toHaveBeenCalledWith(event)
   })
-
-  it('renders tiprack diagrams for selected tipracks', () => {
-    props.values.left = unselectedPipette
```
Comment (shlokamin) [inline protocol-designer/src/components/modals/FilePipettesModal/__tests__/PipetteFields.test.js:162]: No longer a thing

File: protocol-designer/src/components/modals/FilePipettesModal/TiprackDiagram.js [hunk 2]
```diff
@@ -1,37 +0,0 @@
-// @flow
```
Comment (shlokamin) [inline protocol-designer/src/components/modals/FilePipettesModal/TiprackDiagram.js:1]: This component wasn't being used so I deleted it. Idk if we have plans to use this in the future at all though

File: protocol-designer/src/components/modals/EditPipettesModal/index.js [hunk 3]
```diff
@@ -45,7 +44,6 @@ const mapSTP = (state: BaseState): SP => {
     _prevPipettes: stepFormSelectors.getInitialDeckSetup(state).pipettes, // TODO: Ian 2019-01-02 when multi-step editing is supported, don't use initial deck state. Instead, show the pipettes available for the selected step range
     _orderedStepIds: stepFormSelectors.getOrderedStepIds(state),
     thermocyclerEnabled: featureFlagSelectors.getEnableThermocycler(state),
```
Comment (Kadee80) [inline protocol-designer/src/components/modals/EditPipettesModal/index.js:46]: Can see here on github, but did you add `customTipracksEnabled` to the deprecated flags list?

File: protocol-designer/cypress/integration/newProtocol.spec.js [hunk 4]
```diff
@@ -98,15 +98,6 @@ describe('Desktop Navigation', () => {
       cy.get("select[name*='right.tiprack']").select(
         'Opentrons 96 Filter Tip Rack 20 µL'
       )
-      // Diagrams of the tip racks are displayed
-      cy.get("div[class*='FilePipettesModal__tiprack_labware__']")
-        .first()
```
Comment (shlokamin) [inline protocol-designer/cypress/integration/newProtocol.spec.js:103]: no longer a thing

Comment (shlokamin) [reply protocol-designer/cypress/integration/newProtocol.spec.js:103]: @dozercodes is going to add a full e2e covering the custom tiprack flow at a later point

**Review by Kadee80 — APPROVED** [review]
💁

**Review by IanLondon — APPROVED** [review]
:raccoon:

**Pull request merged**